common/test/acceptance/fixtures/course.py

```diff
@@ -152,6 +152,18 @@ def __str__(self):
         """
         return "<CourseFixture: org='{org}', number='{number}', run='{run}'>".format(**self._course_dict)
 
+    def update_course_details(self, course_details):
+        """
+        update course settings (e.g. start and end date)
+
+        Arguments:
+            Dictionary containing key value pairs for course updates,
+            e.g. {'start_date': datetime.now().isoformat() }
+        """
+        self._course_details.update(course_details)
+
+        self._configure_course()
```

Comment:
Can we also make sure here if `course_detail` has `start_date` and `end_date` is converted to iso format as it does inside constructor of `CourseFixture`.